Helpers/UIStylesHelper.m

```diff
@@ -0,0 +1,59 @@
+//
+//  UIStylesHelper.m
+//  social-maps
+//
+//  Created by César Francisco Barraza on 8/6/18.
+//  Copyright © 2018 Bevin Benson. All rights reserved.
+//
+
+#import "UIStylesHelper.h"
+
+@implementation UIStylesHelper
++ (void)setRoundedCornersToView:(UIView *)view {
+    CGFloat width = view.frame.size.width, height = view.frame.size.height;
+    if(width > height)
```

Comment:
you can probably use Math.max(width, height) to figure out which is bigger, then set view.layer.cornerRadius to be max / w